specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/confidentialledger.json

```
@@ -0,0 +1,949 @@
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CollectionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The current ledger entry available on the receiving node. Non-primary nodes may return older values.",
+            "schema": {
+              "$ref": "#/definitions/LedgerEntry"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "common.json#/definitions/ConfidentialLedgerError"
+            }
+          }
+        }
+      }
+    },
+    "/app/users/{userId}": {
+      "delete": {
+        "operationId": "ConfidentialLedger_DeleteUser",
+        "x-ms-examples": {
+          "DeleteUser": {
+            "$ref": "./examples/ledger/DeleteUser.json"
+          }
+        },
+        "summary": "Deletes a user from the Confidential Ledger.",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/UserIdParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "The specified user is deleted."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "common.json#/definitions/ConfidentialLedgerError"
+            }
+          }
+        }
+      },
+      "get": {
+        "operationId": "ConfidentialLedger_GetUser",
+        "x-ms-examples": {
+          "GetUser": {
+            "$ref": "./examples/ledger/GetUser.json"
+          }
+        },
+        "summary": "Gets a user.",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/UserIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Details about the specified user.",
+            "schema": {
+              "$ref": "#/definitions/LedgerUser"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "common.json#/definitions/ConfidentialLedgerError"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "ConfidentialLedger_CreateOrUpdateUser",
+        "x-ms-examples": {
+          "CreateOrUpdateUser": {
+            "$ref": "./examples/ledger/CreateOrUpdateUser.json"
+          }
+        },
+        "summary": "Adds a user or updates a user's fields.",
+        "description": "A JSON merge patch is applied for existing users",
```

A patch operation should consume 'application/merge-patch+json' content type.
```suggestion
        "description": "A JSON merge patch is applied for existing users",
        "consumes": ["application/merge-patch+json"],
```